fontbe/src/features.rs

```diff
@@ -434,7 +434,10 @@ impl FeatureCompilationWork {
 }
 
 fn write_debug_glyph_order(context: &Context, glyphs: &GlyphOrder) {
-    let glyph_order_file = context.debug_dir().join("glyph_order.txt");
+    let Some(debug_dir) = context.debug_dir() else {
+        return;
```

Comment:
If the invariant is that EMIT_DEBUG being set guarantees debug_dir() returns Some, then reaching the None case is a bug that should fail fast, not silently continue. I'd say we should `.expect("debug_dir must exist when EMIT_DEBUG is set")` here